web-platform/website/src/services/AuthenticationService.tsx

```diff
@@ -109,6 +112,13 @@ export function AuthenticationProvider(props: { children: React.ReactNode }) {
         isLoaded,
         isLoggingIn,
         isUserVerified,
+        reloadUser: async () => {
+          if (!user || !user.firebaseUser) {
+            return;
+          }
+          await user.firebaseUser.reload();
+          updateUserVerificationState(user.firebaseUser);
+        },
```

Comment:
We can keep this private in this file and do it within useEffect(() => {}, []). Ideally this should only be done once at the first load.

Reply:
AFAICT this doesn't work because useEffect does not run when user clicks "Skip for now" on the get extension screen. We need to expose some way to manually refresh the user.

Ideally "onAuthStateChanged" would fire when the user verifies their email (which would avoid all this) but that is not the case (and lots of people on StackOverflow have complained about it)